### PR TITLE
[yaml/ko-kr]Translate YAML doc

### PR DESCRIPTION
**Pull request by linterpreteur**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
- [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: ko-kr/yaml-kr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+# 파이썬과 마찬가지로 집합은 단지 널 값을 갖는 맵입니다. 위는 다음과 같습니다.
+집합2:
+    하나: null
+    둘: null
+    셋: null
+```
+
+### 더 읽기
+
++ [YAML 공식 사이트](http://yaml.org/)
```
Comment (vendethiel) [inline ko-kr/yaml-kr.html.markdown]: please add " (EN)" tags or use ko-kr links. LGTM otherwise!

Comment (linterpreteur) [inline ko-kr/yaml-kr.html.markdown]: None of the existing translations have such notations. Wouldn't that be against uniformity?

Comment (vendethiel) [inline ko-kr/yaml-kr.html.markdown]: It's fine; they'll get updated at some point

Comment (linterpreteur) [inline ko-kr/yaml-kr.html.markdown]: Does it mean every link will eventually marked i.e. (EN) if it is in English, and therefore I need to do so to make changes that I've made merged to master?

Comment (vendethiel) [inline ko-kr/yaml-kr.html.markdown]: It will eventually, yes (well, unless they're korean). No rush, though.

Comment (linterpreteur) [inline ko-kr/yaml-kr.html.markdown]: I guess (영어) would make more sense than (EN) because this targets Korean speaking readers.

**Comment by linterpreteur** [comment]
@vendethiel Please review the update, as well as ones of other PRs of mine:

* [XML](https://github.com/adambard/learnxinyminutes-docs/pull/2487)
* [Markdown](https://github.com/adambard/learnxinyminutes-docs/pull/2486)
* [JSON](https://github.com/adambard/learnxinyminutes-docs/pull/2485)

Thanks.

File: ko-kr/yaml-kr.html.markdown (hunk 1 above)
Comment (vendethiel) [inline ko-kr/yaml-kr.html.markdown]: Please do :)

Comment (linterpreteur) [inline ko-kr/yaml-kr.html.markdown]: Does this PR still need any improvements?

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**